api/src/main/resources/application-cloud.properties

```diff
@@ -15,6 +15,4 @@
 # limitations under the License.
 #
 
-spring.h2.console.enabled=false
```

Comment:
I think we still do want the console disabled in the cloud version of the properties file; why not just enable console in application.properties?

Reply:
Good catch. I will move it to the application.properties file to make it the default.